client/src/app/site/users/models/view-user.ts

```diff
@@ -89,6 +89,21 @@ export class ViewUser extends BaseProjectableViewModel<User> implements Searchab
         return this.is_present_in_meeting_ids?.includes(meetingId);
     }
 
+    public get hasMultipleMeetings(): boolean {
+        return this.user.is_present_in_meeting_ids?.length !== 1;
+    }
+
+    public get onlyMeeting(): Id {
+        const meetingAmount = this.user.is_present_in_meeting_ids?.length || 0;
+        if (meetingAmount === 1) {
+            return this.user.is_present_in_meeting_ids[0];
+        } else if (meetingAmount > 1) {
+            throw new Error(`User has multiple meetings`);
+        } else if (meetingAmount === 0) {
+            throw new Error(`User has no meetings at all`);
+        }
+    }
+
```

Comment:
I know, this PR is still "work in progress" and perhaps you have that already in mind, but one hint:
`user.is_present_in_meetings` refers only to the meetings a user is present in. This means, there could be meetings, a user has access to, but is currently not as "present" marked.